runtime/compiler/control/CompilationThread.cpp

```diff
@@ -6867,8 +6880,16 @@ TR::CompilationInfoPerThreadBase::preCompilationTasks(J9VMThread * vmThread,
       //
       TR::IlGeneratorMethodDetails & details = entry->getMethodDetails();
 
-      eligibleForRelocatableCompile =
-            (enableJITaaSRemoteAOT || !entry->isOutOfProcessCompReq()) && // disable remote JITaaS AOT until it's working
+      if (_compInfo.getPersistentInfo()->getJITaaSMode() == SERVER_MODE)
```

Comment:
rather than testing `_compInfo.getPersistentInfo()->getJITaaSMode() == SERVER_MODE` it's nicer to test entry->isOutOfProcessCompReq()